main.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable max-classes-per-file */
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const add = document.getElementById('add');
+const list = document.getElementById('listTodo');
+let currendID = Date.now();
+class Books {
+  constructor(title, author = null, id) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class ListBooks {
+  constructor() {
+    this.arrayOfTitle = [];
+  }
+
+  // Add task to array
+ addTask = (title, author) => {
+   const book = new Books(title, author, currendID);
+   this.arrayOfTitle.push(book);
+ }
+
+  deleteFromLocal = (taskId) => {
+    /* eslint-disable eqeqeq */
```

Comment:
- [ ] Kindly remove this, as it is decapitating linters from performing and a standard syntax check on your code.
Don't forget that organizations use this to enforce coding standards in their organization, and since your will be part of one soon, I think you should get used to mastering linting styles :smile: :muscle: